### DISCUSSION TITLE
UBOL and the built-in adblocking feature of the browser

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBOL-home#126] [General]
Chrome has had a built-in adblocking feature since several versions ago. Do I need to disable it to use UBOL?

**Comment by garry-ut99** [comment]
Yes you need. You should never use two ad-blockers at the same time, because it might cause breakages.
Read FAQ before asking, as that question is answered already: 
- https://github.com/uBlockOrigin/uBOL-home/wiki/Frequently-asked-questions-(FAQ)#can-i-use-other-content-blockers-along-ubo-lite
- https://github.com/uBlockOrigin/uBOL-home#frequently-asked-questions-faq
- https://github.com/gorhill/uBlock/?tab=readme-ov-file#all-programs (on uBO repo)
- https://github.com/uBlockOrigin/uAssets?tab=readme-ov-file#similarly-purposed-blockers (on uAssets tracker)

**Reply by ghost** [reply]
These pages are mentioning about ad-blocker extensions. I'm talking about the built-in ad-blocking feature of Chrome (it's part of the browser and not an extension).

**Reply by garry-ut99** [reply]
Wiki page mentions extensions, but Twitter page does not and it only mentions "similiar-purposed blocker(s)", anyway the context of both is most likely: to not use two or more "similar-purposed blocker(s)", regardless of whether any of them is built-in or not, because both of them might interfere with each other and/or break/slowdown pages, unless you are an advanced user and you exactly know what you are doing and/or you want to take a risk, of course nothing stops you from trying and testing, and if nothing bad happens, you might try to stay with both of them.

**Comment by ghost** [comment]
The build-in Chrome ad block is enabled only for some pages. I think I saw it only on one page ever. You will see a message/icon in address bar when this happens. I don't think you should worry too much about it. Unless you have some real world problem with it right now?

**Reply by garry-ut99** [reply]
But some of these sites might be popular sites, visited often or every day, also different people visit different sites, hence there always is the risk, the best if he just tries out himself, enables both of them for few days and see whether there are any problems.
